coco/master.py

```diff
@@ -138,15 +147,16 @@ def __del__(self):
 
         Join the worker process.
         """
-        logger.info("Joining worker process...")
-        try:
-            self.redis_sync.rpush("queue", "coco_shutdown")
-        except BaseException as e:
-            logger.error(
-                f"Failed sending shutdown command to worker (have to kill it): {type(e)}: {e}"
-            )
+        if not self.check_config:
```

Comment:
This may not even be necessary because it seems `_kill_worker` checks if the worker exists before it kills. =) But nice catch!

Reply:
Yeah, but it would still write some log lines that don't make much sense in this mode.